packages/components/src/range-control/styles/range-control-styles.js

```diff
@@ -0,0 +1,310 @@
+/**
+ * External dependencies
+ */
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { color } from '../../utils/colors';
+
+const rangeHeight = () => css( { height: 30, minHeight: 30 } );
+const rootRtl = ( { isRTL } ) => css( { direction: isRTL ? 'rtl' : 'ltr' } );
+
+export const Root = styled.span`
+	-webkit-tap-highlight-color: transparent;
+	box-sizing: border-box;
+	cursor: pointer;
+	align-items: flex-start;
+	display: inline-flex;
+	justify-content: flex-start;
+	padding: 0;
+	position: relative;
+	touch-action: none;
+	width: 100%;
+
+	${rootRtl}
+`;
+
+const wrapperMargin = ( { marks } ) =>
+	css( { marginBottom: marks ? 16 : null } );
+
+export const Wrapper = styled.span`
+	box-sizing: border-box;
+	color: ${color( 'blue.medium.focus' )};
```

Comment:
I see several cases of "blue" usage in this file. Do these adapt to the WordPress admin scheme colors? Should they?
we have the `theme(something)` colors in SCSS to solve that problem there.

Reply:
@youknowriad These colours correspond to the [(SCSS) based colour values](https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/_colors.scss), but are retrieved via the `color()` util function. The original mappings are here:

https://github.com/WordPress/gutenberg/blob/master/packages/components/src/utils/colors-values.js#L76

Reply:
but what is the expectation here, do we want these colors to adapt to WordPress's admin scheme colors or not?